WEATHERAPP/Components/Login/LoginPage.css

```diff
@@ -0,0 +1,112 @@
+body
+{
+   margin: 0;
+}
+.background
+{
+   height: 100vh;
+   background-image:url("../../Assets/L&D-BG\ 4.png");
```

Comment:
Add background-size: cover and background-position: center to ensure the image covers the entire container.

Reply:
resolved